src/main/java/com/org/jenkins/custom/jenkins/distribution/service/GithubController.java

```diff
@@ -0,0 +1,56 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import java.util.List;
+import org.kohsuke.github.GHApp;
+import org.kohsuke.github.GHAppInstallation;
+import org.kohsuke.github.GHAppInstallationToken;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.org.jenkins.custom.jenkins.distribution.service.GithubJwtHelper.createJWT;
+import static com.org.jenkins.custom.jenkins.distribution.service.GithubJwtHelper.get;
+
+
+@RestController
+@CrossOrigin("*")
+public class GithubController {
+
+    @Value("${APP_IDENTIFIER}")
```

Comment:
The value is read from the properties file. Does the user need to change this value when they run the application?

Reply:
If this is hosted no, if it is self-hosted then yes

Reply:
In the case where it is self-hosted, the user needs a way to change the `APP_IDENTIFIER` value then. I would recommend finding a way to read this value from an environment variable, or a user provided properties file, without having the user change the source code of the application.